src/KeyVault/KeyVault/help/Initialize-AzKeyVaultSecurityDomainRecovery.md

```diff
@@ -0,0 +1,203 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.KeyVault.dll-Help.xml
+Module Name: Az.KeyVault
+online version: https://learn.microsoft.com/powershell/module/az.keyvault/initialize-azkeyvaultsecuritydomainrecovery
+schema: 2.0.0
+---
+
+# Initialize-AzKeyVaultSecurityDomainRecovery
```

Comment:
Could you provide operation id used by this command? Wanna discuss the naming of this cmdlet.